sdk/tests/utilities/test_api_client_factory.py

```diff
@@ -112,6 +112,14 @@ def test_get_api_with_configuration(self):
         self.assertIsInstance(api, InstrumentsApi)
         self.validate_api(api)
 
+    def test_get_api_with_pat_env_var(self):
+        with patch.dict('os.environ', {"FBN_PASSWORD": "", "FBN_CLIENT_SECRET": ""}, clear=False):
+            factory = ApiClientFactory()
+            api = factory.build(InstrumentsApi)
+            self.assertIsInstance(api, InstrumentsApi)
+            self.validate_api(api)
+            self.assertEqual(factory.auth_method, "PersonalAccessToken")
```

Comment:
Test that the API Factory will default to the PAT in the env variables when there is no secrets file or token passed as a param.